tests/integration/test_sync.py

```diff
@@ -94,25 +94,6 @@ def test_sync_sequential_detect_errors(PipenvInstance):
         assert c.returncode != 0
 
 
-@pytest.mark.sync
-@pytest.mark.lock
-def test_sync_sequential_verbose(PipenvInstance):
```

Comment:
The verbose output is driven by the output of pip, no sense having a test depend on what it or what it did before since pip is subject to future upgrades.